components/questions.js

```diff
@@ -44,7 +45,16 @@ export default function Questions(props) {
 
   function handleClick(next) {
     if (next && confidence === DEFAULT_CONFIDENCE) {
-      setError(true);
+      setError("error1");
```

Comment:
Please use more more descriptive names so that the code is easier to maintain, for example here `noConfidence`